coralproject/talk
# Post comment

## What does this PR do?

Fixes a bug in comment count, fixes post order.

## How do I test this PR?

1. `npm run embed-start`
2. Comment count should include replies.
3. Comments should appear in chronological order when posted.
4. Comments should appear in chronological order on page reload.
5. Pym.js in `http://localhost:3000/client/coral-embed-stream/samplearticle.html` should serve over https.

@coralproject/tech
